javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,4 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  return values.filter(Boolean);
```

Comment:
can you implement your own logic here with a loop instead? This is also technically going to filter out any `falsy` value, not just `null`